package.json

```diff
@@ -10,6 +10,14 @@
     "test": "jest"
   },
   "dependencies": {
+
```

Comment:
Remove whiteline here as well. Better use code formatter like Prettier to format and structure codes on saving.